unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -2116,6 +2116,92 @@ makeStandalone =
         _ -> Left $ showPatternHelp makeStandalone
     )
 
+runScheme :: InputPattern
+runScheme =
+  InputPattern
+    "run.scheme"
```

Comment:
Yeah, I actually figure out how to do this first, because when you call the scheme binary with a program file, it just runs it. To compile it, you need to actually start the repl and feed it commands to compile your stuff, so it's more complicated.